### PR TITLE
Add total hits relation in ResultSelt

### PR DESCRIPTION
**Pull request by deguif**

### CODE REVIEW

**Comment by ruflin** [comment]
Could you add a changelog entry and if possible add a small functional tests?

**Comment by deguif** [comment]
@ruflin @thePanz tests and changelog entry were added

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 
 * support for elasticsearch-php ^7.0
+* Added `\Elastica\ResultSet::getTotalHitsRelation()` to get relation for total hits [#1694](https://github.com/ruflin/Elastica/pull/1694)
```
Comment (thePanz) [inline CHANGELOG.md]: This line should be added to the "unreleased" changes, after line 10!

Comment (deguif) [reply CHANGELOG.md]: Oups, I messed up with the previous changes, I fixed that

Comment (deguif) [reply CHANGELOG.md]: @ruflin @thePanz do you think that can be merged now, as I will have to rebase my other PR as I changed many lines the changelog here

**Comment by ruflin** [comment]
@deguif Merged. Thanks for also cleaning up the other changelog entries, miss that in the PR review :-(

**Pull request merged**